src/Actions/actions.js

```diff
@@ -0,0 +1,69 @@
+import axios from 'axios';
+import actionTypes from '../Actions/actionTypes';
+import 'regenerator-runtime/runtime'
+
+const {
+    GET_MOVIES,
+    GET_MOVIE,
+    UPDATE_MOVIE,
+    ADD_MOVIE,
+    DELETE_MOVIE,
+    SORT_MOVIES,
+    FILTER_MOVIES,
+} = actionTypes;
+
+export const getMovies = () => async dispatch => {
+    const res = await axios.get('http://localhost:4000/movies')
```

Comment:
i think it's a good idea to move it to constant
```js
const API = 'http://localhost:4000/'
```

Reply:
fixed